SampleViewer/SampleViewer.pro

```diff
@@ -271,11 +271,11 @@ CONFIG(daily){
     CONFIG(release, debug | release){
         win32 {
             contains(QT_ARCH, i386) {
-                message($${PWD}/../../../scripts/windows/deploy_windows.bat -bin $${DESTDIR} -dep ArcGISRuntimeSDKQt_CppSamples -exe ArcGISQt_CppSamples.exe -qt $${QMAKESPEC}/../.. -b x86 -toolkit -tk_qml)
-                QMAKE_POST_LINK +=$$quote(cmd /c $${PWD}/../../../scripts/windows/deploy_windows.bat -bin $${DESTDIR} -dep ArcGISRuntimeSDKQt_CppSamples -exe ArcGISQt_CppSamples.exe -qt $${QMAKESPEC}/../.. -b x86 -toolkit -tk_qml $$escape_expand(\n\t))
+                message($${PWD}/../../scripts/windows/deploy_windows.bat -bin $${DESTDIR} -dep SampleViewer -exe ArcGISQt_Samples.exe -qt $${QMAKESPEC}/../.. -b x86 -toolkit -tk_qml)
+                QMAKE_POST_LINK +=$$quote(cmd /c $${PWD}/../../scripts/windows/deploy_windows.bat -bin $${DESTDIR} -dep SampleViewer -exe ArcGISQt_Samples.exe -qt $${QMAKESPEC}/../.. -b x86 -toolkit -tk_qml $$escape_expand(\n\t))
```

Comment:
@JamesMBallard We don't have a 32bit build anymore. Do we still need this? Also looking at the original I don't understand why the -dep argument is different between the two.
See original: https://github.com/Esri/arcgis-maps-sdk-samples-qt/blob/0a84848c8fd1621b532190a2f6436213c719689e/ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_CppSamples/ArcGISRuntimeSDKQt_CppSamples.pro#L272-L279

Reply:
No, you can simplify this. We haven't supported 32-bit Windows for a while now.